src/scripts/lib/server/utils/server-config/index.js

```diff
@@ -0,0 +1,26 @@
+const Configstore = require('configstore')
+const inq = require('inquirer')
+const prompts = require('./setup-prompts')
+
+module.exports = class ServerConfig {
+
+  constructor(defaults) {
+    this.db = new Configstore('wowser', defaults)
```

Comment:
I tried that, but once you run `gulp` the directory context changes since the lib is now in the root and not in `src`. Not sure how to work around that.